CMakeLists.txt

```diff
@@ -2,16 +2,16 @@ cmake_minimum_required(VERSION 3.1.3)
 
 project(Fast_BVH LANGUAGES CXX)
 
-option(FASTBVH_BUILD_EXAMPLES "Whether or not to build the example programs." ON)
-option(FASTBVH_BUILD_TESTS    "Whether or not to build the unit tests." ON)
+option(FASTBVH_BUILD_EXAMPLES  "Whether or not to build the example programs." ON)
+option(FASTBVH_BUILD_TESTS     "Whether or not to build the unit tests." ON)
 option(FASTBVH_BUILD_BENCHMARK "Whether or not to build the benchmark." OFF)
 
 set(CMAKE_CXX_STANDARD 14)
 
 set(gnu_cxxflags -Wall -Wextra -Werror -Wfatal-errors)
 
 if(CMAKE_BUILD_TYPE STREQUAL "Release")
-  list(APPEND gnu_cxxflags -ffast-math)
+  list(APPEND gnu_cxxflags -ffast-math -march=native -mtune=native)
```

Comment:
Leads to better auto-vectorization on platforms supporting AVX2. In my experience, most CPUs support this, although it's not enabled by GCC or Clang unless either these options detect it or it's manually turned on with `-mavx2`.